dev/create-release/release-build.sh

```diff
@@ -446,6 +432,8 @@ if [[ "$1" == "publish-release" ]]; then
   # Clean-up Zinc nailgun process
   $LSOF -P |grep $ZINC_PORT | grep LISTEN | awk '{ print $2; }' | xargs kill
 
+  ./dev/change-scala-version.sh 2.11
```

Comment:
To make the code cleaner, I'd suggest we change the scala version back right after we finish building scala 2.10 or 2.12, e.g.
```
... build with scala 2.11

if PUBLISH_SCALA_2_10 = 1
  ./dev/change-scala-version.sh 2.10
  ... build with scala 2.10
  ./dev/change-scala-version.sh 2.11

if PUBLISH_SCALA_2_12 = 1
  ./dev/change-scala-version.sh 2.12
  ... build with scala 2.12
  ./dev/change-scala-version.sh 2.11
```

cc @srowen

Reply:
I guess it couldn't hurt to carefully change it back right after each step is done that needs a different version. As long as it probably happens in the end no matter what.

I am fine with the other changes.